package.json

```diff
@@ -38,6 +34,14 @@
     "webpack-node-externals": "^1.7.2"
   },
   "engines": {
-    "node": "12.x"
+    "node": "12.x",
+    "npm": "7.x || 8.x"
+  },
+  "husky": {
```

Comment:
I think this husky bit has been done automatically. Let me know if I should revert it. The scripts seem to be running anyways.

Reply:
ok